libvast/src/start_command.cpp

```diff
@@ -147,6 +144,9 @@ caf::message start_command(const invocation& inv, caf::actor_system& sys) {
       self->send(runner, atom::run_v, *hook_invocation);
     }
   }
+  // A single line of output to publish our address for scripts.
+  if (caf::get_or(inv.options, "tenzir.start.print-endpoint", false))
+    std::cout << listen_addr << std::endl;
```

Comment:
This must move back up to where it was before. We run the start commands asynchronously. For example:

```
tenzir-node --commands="exec version" --print-endpoint
```

This prints the version and the port in a non-deterministic order.

If you move it back up then the endpoint is at least guaranteed to be printed first.